### ISSUE TITLE
Getting "javadoc: error - invalid flag: -html4" when trying to build

### ISSUE THREAD

**Issue by amnox** [grpc/grpc-java#7637]
### What version of gRPC-Java are you using?
I am on master branch. git status gives this

```
On branch master
Your branch is up to date with 'origin/master'.
```
### What is your environment?

I am using  ubuntu 18.04
```
openjdk 11.0.9.1 2020-11-04
OpenJDK Runtime Environment (build 11.0.9.1+1-Ubuntu-0ubuntu1.18.04)
OpenJDK 64-Bit Server VM (build 11.0.9.1+1-Ubuntu-0ubuntu1.18.04, mixed mode, sharing)
```
Gradle
```
------------------------------------------------------------
Gradle 5.6.2
------------------------------------------------------------

Build time:   2019-09-05 16:13:54 UTC
Revision:     55a5e53d855db8fc7b0e494412fc624051a8e781

Kotlin:       1.3.41
Groovy:       2.5.4
Ant:          Apache Ant(TM) version 1.9.14 compiled on March 12 2019
JVM:          11.0.9.1 (Ubuntu 11.0.9.1+1-Ubuntu-0ubuntu1.18.04)
OS:           Linux 4.15.0-123-generic amd64

```
### What did you expect to see?
I was following steps in [building grpc-java](https://github.com/grpc/grpc-java/blob/master/COMPILING.md), with all prerequisites (including protobuf) and expected a successful build using gradle.
### What did you see instead?
While trying to `./gradlew build` with skip android in `gradle.properties` I got error
```
javadoc: error - invalid flag: -html4
```
### Steps to reproduce the bug
Just run `./gradlew build`

### How did I fix it
I commented out line 282 in `build.gradle`
```
if (JavaVersion.current().isJava10Compatible()) {
        allprojects {
            tasks.withType(Javadoc) {
=======>>>   //options.addBooleanOption('html4', true)
            }
        }
    }

```
I know this is not the ideal way to do this. Can anyone provide more information on this? Thanks!

**Comment by dapengzhang0** [comment]
As commented here https://github.com/grpc/grpc-java/blob/v1.33.1/build.gradle#L278,  we must specify one html version for java 10. For java 11+, 'html4' option is removed. So we may choose 'html5' option to support all jdk version.

**Comment by amnox** [comment]
I tried to change that option to html5 aswell but got same error.

**Comment by dapengzhang0** [comment]
> I tried to change that option to html5 aswell but got same error.

Then seems both options are removed in newer jdk. So we might as well remove line 282 in `build.gradle` and we don't care too much about java 10 because it is not a long term supported version.

**Comment by dapengzhang0** [comment]
> Can I submit a PR for this?

Yes, please!

**Comment by amnox** [comment]
Sure, submitting it now.

**Comment by amnox** [comment]
I have created the PR #7645

**Comment by dapengzhang0** [comment]
Fixed by #7645. Thank you @amnox
